Solutions/AIShield AI Security Monitoring/Parsers/AIShield.yaml

```diff
@@ -1,22 +1,23 @@
 id: 574a5c4d-051c-41c8-83a9-f06962e14d85
 Function:
   Title: Parser for AIShield
-  Version: '1.0.0'
-  LastUpdated: '2023-08-23'
+  Version: '1.0.1'
+  LastUpdated: '2023-12-21'
 Category: Microsoft Sentinel Parser
 FunctionName: AIShield
 FunctionAlias: AIShield
 FunctionQuery: |
     AIShield_CL
     | extend EventVendor = 'Bosch'
     | extend EventProduct = 'AIShield'
-    | extend Severity = iff(probability_d between (0.85..1.0),"High",iff(probability_d between
-      (0.7..0.84), "Medium", iff(probability_d < (0.7),"Low","Unknown")))
+    | extend Severity = iff(probability_d > 0.50 and suspicious_level_s =~ 'high suspicious attack',"High",iff(probability_d between (0.0..0.50) and suspicious_level_s =~ 'high suspicious attack',"Low",iff(probability_d between (0.0..0.50) and suspicious_level_s =~ 'low suspicious attack',"Information",iff(probability_d between (0.50..0.75) and suspicious_level_s =~ 'low suspicious attack',"Low",iff(probability_d between (0.75..1) and suspicious_level_s =~ 'low suspicious attack',"Medium","unknown")))))
     | project-rename
         AttackName = attack_name_s,
         Message = RawMessage_s,
         ServiceName = service_name_s,
         AssetId = asset_id_s,
         EventTimestamp = timestamp_t,
-        SourceName = source_name_s
+        SourceName = source_name_s,
+        SuspiciousLevel = suspicious_level_s,
+        Probability = probability_d
     | project-away probability_d
```

Comment:
Hello @udanashivm, project-away should be column Probability. as column probability_d is renamed to Probability

Reply:
I'll commit this change, probability_d is renamed to Probability. As this is causing parser failure.

Reply:
Thank you for the approval & merge @v-prasadboke  :)